src/bin/bios.rs

```diff
@@ -1,6 +1,4 @@
 #![feature(lang_items)]
```

Comment:
Is this still necessary?

Reply:
All tests passed even without it. I'm just wondering that I did not get a warning beforehand. I updated the commit

Reply:
Unused feature flags haven't been given a warning ever since the introduction of incremental compilation. Only now stabilized feature flags get a warning. `lang_items` is still unstable, but it seems that the current code doesn't use them anymore.

Reply:
Got it. Thanks for the explanation!

Reply:
@bjorn3 Good catch!